backend/src/db-utils.js

```diff
@@ -356,6 +356,68 @@ const aggregateStats = async (options) => {
   };
 };
 
+const queryTransactionsByDate = async () => {
```

Comment:
@khorolets This `queryTransactionsByDate` misleads the reader to assume that it can be used to query a list of transactions by a given date. I suggest we rename it to something like `queryTransactionsCountAggregatedByDate`